### PR TITLE
Seanrester/sc 48196/update serial regression tests to run in

### PR DESCRIPTION
**Pull request by syntastical**
#### What this PR does / why we need it:
Adds async regression testing and cleanup

#### Which issue(s) this PR fixes:

Fixes #48196

#### Special notes for your reviewer:
NONE

## Steps to reproduce
NONE

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Does this PR require documentation?
NONE

### CODE REVIEW

File: actions/clean-up/dist/index.js [hunk 1]
```diff
@@ -0,0 +1,3537 @@
+/******/ (() => { // webpackBootstrap
```
Comment (emosbaugh) [inline actions/clean-up/dist/index.js:1]: was it intentional to include this file?

File: actions/clean-up/index.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+      },
+      ignoreReturnCode: true,
+      cwd: 'automation/jumpbox'
+    });
+
+  if(completionTimeRaw) {
+    const currentTime = new Date();
+    const completionTime = new Date(completionTimeRaw);
+    // if(currentTime.getTime() - completionTime.getTime() > (1000 * 60 * 60 * 24)) {
+    if(currentTime.getTime() - completionTime.getTime() > (1)) {
```
Comment (emosbaugh) [inline actions/clean-up/index.js:94]: is this debug code? what is the intention here?

**Review by emosbaugh — APPROVED** [review]

File: actions/clean-up/index.js (hunk 2 above)
Comment (syntastical) [reply actions/clean-up/index.js:94]: Ooops yeah that should be changed

File: actions/clean-up/dist/index.js (hunk 1 above)
Comment (syntastical) [reply actions/clean-up/dist/index.js:1]: Yeah, this file is generated with the dependencies, so we don't need to check in node_modules.

**Review by emosbaugh — APPROVED** [review]

**Review by emosbaugh — APPROVED** [review]

**Pull request merged**